spin.toml

```diff
@@ -232,6 +233,15 @@ environment = { DESTINATION = "https://developer.fermyon.com/spin/ai-sentiment-a
 route = "/spin/serverless-ai-tutorial"
 executor = { type = "wagi" }
 
+# Redirect /cloud/serverless-ai to /spin/serverless-ai-hello-world
+[[component]]
+id = "serverless-ai"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/serverless-ai-hello-world.md" }
+[component.trigger]
+route = "/spin/serverless-ai-hello-world"
```

Comment:
This will redirect spin/serverless-ai-world to itself which I don't think is what you want.  The route is the page you want to redirect _from_ (I _think `/cloud/serverless-ai`); the environment DESTINATION is where to redirect it _to_.

Reply:
It's not that it's cached or preserved - it's that the redirector sees the request before the CMS does.  The CMS _would_ care that the .md file no longer existed; but the redirector prevents requests for that old file ever reaching the CMS.  Thanks for tuning into today's episode of "More Than You Ever Cared About Spin Redirection Internals," don't forget to mash that subscribe button.